client/layout/sidebar/style.scss

```diff
@@ -277,3 +277,16 @@
 .sidebar__menu-wrapper {
 	border-top: 1px solid var( --color-sidebar-border );
 }
+
+.sidebar__menu-link-badge {
+	display: inline-block;
+	padding: 1px 6px;
+	border: solid 1px var( --studio-yellow );
```

Comment:
I don't think we want `--studio-yellow` here. To match the screenshots we probably want `--color-warning`, and maybe not colorizing the background fully. I'd try:

```
border: 1px solid var( --color-warning );
color: var( --color-warning-light );
```

And no `background-color`, or maybe `--color-accent-dark`. I've pinged @ianstewart in Slack to see what he thinks.

Reply:
No to the background, after checking with Ian. With just border and color like above, it should look like:  
<img width="178" alt="Screen Shot 2021-07-20 at 12 28 54" src="https://user-images.githubusercontent.com/195089/126376354-756dc004-bdf7-478c-9d6b-2df82dc7ec30.png">

Reply:
Thanks! Updated the colors